solr/solrj/src/test/org/apache/solr/client/solrj/response/TermsResponseTest.java

```diff
@@ -31,16 +35,19 @@ public class TermsResponseTest extends EmbeddedSolrServerTestBase {
 
   @BeforeClass
   public static void beforeClass() throws Exception {
-    initCore();
+    solrClientTestRule.startSolr(LuceneTestCase.createTempDir("solrhome"));
+
+    solrClientTestRule
+        .newCollection(DEFAULT_TEST_COLLECTION_NAME)
+        .withConfigSet(ExternalPaths.TECHPRODUCTS_CONFIGSET)
+        .create();
```

Comment:
Exemplifies we're not after shaving lines of code.  Starting Solr and creating a collection with a known configSet is explicit and should be -- isn't verbose either.